docker/docker-compose.yml

```diff
@@ -10,15 +10,43 @@ services:
         source: ${PWD}/vmpooler.yaml
         target: /etc/vmpooler/vmpooler.yaml
     ports:
-      - "4567:4567"
+      - "8080:4567"
     networks:
       - redis-net
     environment:
       - VMPOOLER_DEBUG=true # for use of dummy auth
       - VMPOOLER_CONFIG_FILE=/etc/vmpooler/vmpooler.yaml
       - REDIS_SERVER=redislocal
       - LOGFILE=/dev/null
+      - JRUBY_OPTS=-Xinvokedynamic.yield=false
+      - VMPOOLER_TRACING_ENABLED=true
+      - VMPOOLER_TRACING_JAEGER_HOST=http://jaeger-aio:14268/api/traces
```

Comment:
What happens for someone outside of puppet if running with tracing enabled and this host target?

Reply:
That target is the one in docker compose so it's 100% local.